loky/process_executor.py

```diff
@@ -673,14 +672,14 @@ def is_shutting_down():
             _shutdown_crash(executor_flags, processes, pending_work_items,
                             call_queue, cause_msg)
             return
-        executor = executor_reference()
         if (is_shutting_down() and
```

Comment:
You are right. But the behavior is still okay now because `getattr(queue_management_thread, "_clean_exit", False)` is `True` only when the executor is shutting down so I propose to remove `is_shutting_down`